spec/index.html

```diff
@@ -297,15 +297,48 @@ <h3>IRIs</h3>
     <h3>Literals</h3>
 
     <p><a data-cite="RDF12-CONCEPTS#dfn-literal">Literals</a>
-    are basic values that are not IRIs. Examples of literals include
-    strings such as "La Joconde", dates such as "the 4th of July, 1990"
-    and numbers such as "3.14159".
-    Literals are associated with a <a data-cite="RDF12-CONCEPTS#dfn-datatype">datatype</a>
-    enabling such values to be parsed and interpreted correctly.
-    String literals can optionally be associated with a <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>.
-    For example, "Léonard de Vinci" could
-    be associated with the "fr" language tag and "李奥纳多·达·文西"
-    with the "zh" language tag.</p>
+    are used to encode values, rather than identifying them by IRIs. Examples
+    of literals include strings such as "La Joconde", dates such as
+    "1990-07-04" (the 4th of July, 1990) and numbers such as "3.14159".
+    Literals are always associated with a
+    <a data-cite="RDF12-CONCEPTS#dfn-datatype">datatype</a>, enabling them to
+    be parsed and interpreted correctly.</p>
+
+    <p>Strings can optionally be associated with a <a
+        data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>.
+    For example, to clarify that the name "Leonardo da Vinci" is the English
+    verison of his name, the string can be associated with the <code>en</code>
+    language tag, "Léonard de Vinci" with the <code>fr</code> language tag (French),
+    and "李奥纳多·达·文西" with the <code>zh</code> language tag (Chinese).
+    Such literals are called
+    <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string">language-tagged strings</a>.
+    </p>
+
+    <p>Strings may also optionally be associated with an initial <a>base
+    direction</a>, with a value of either <code>ltr</code> or <code>rtl</code>
+    (for left-to-right and right-to-left, respectively). For example, the English
+    title <q lang="en" dir="ltr">HTML &amp; CSS: Designing and Creating Websites</q>,
+    associated with the <code>en</code> language tag and recognised as written
```

Comment:
W3C generally wants [US English](https://www.w3.org/guide/manual-of-style/#Spelling).
```suggestion
    associated with the <code>en</code> language tag and recognized as written
```